content/id/docs/reference/glossary/persistent-volume-claim.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Persistent Volume Claim
+id: persistent-volume-claim
+date: 2018-04-12
+full_link: /docs/concepts/storage/persistent-volumes/
```

Comment:
I am not sure if it s need to translate to ID. But we can do it later.

Reply:
I think we need to update this, forgot that line. When you click a glossary, if it has a link (like this one), it should be redirected there.